test-cypress.sh

```
@@ -30,18 +30,20 @@ do
 # Run tests.
-yarn_script="test-cypress-headless"
+npm_script="test-cypress-headless"
 
 if [ -n "${gui-}" ]; then
-  yarn_script="test-cypress"
+  npm_script="test-cypress"
 fi
 
 if [ -n "${spec-}" ]; then
-  yarn_script="$yarn_script --spec '$spec'"
+  npm_script="$npm_script --spec '$spec'"
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Use an array.

(SC2089)